packages/lib-classifier/src/components/Classifier/Classifier.js

```diff
@@ -49,6 +49,8 @@ export default function Classifier({
   useEffect(function onWorkflowVersionChange() {
     const { workflows, subjects } = classifierStore
     if (workflowSnapshot) {
```

Comment:
```suggestion
    if (workflowSnapshot && workflows?.active) {
```

If I don't edit as noted then I get `Uncaught TypeError: Cannot read properties of undefined (reading 'subjectSet')` when testing on https://local.zooniverse.org:8080/?project=nora-dot-eisner/planet-hunters-tess&env=production&demo=true